upkie/cpp/actuation/BulletInterface.h

```diff
@@ -31,17 +32,23 @@ class BulletInterface : public Interface {
     //! Keep default constructor.
     Parameters() = default;
 
+    //! Placeholder at initization time.
+    std::map<std::string, int> empty_map = {{"", 0}};
     /*! Initialize from global configuration.
      *
      * \param[in] config Global configuration dictionary.
      */
-    explicit Parameters(const Dictionary& config) { configure(config); }
+    explicit Parameters(const Dictionary& config) {
+      configure(config, empty_map);
+    }
 
     /*! Configure from dictionary.
      *
      * \param[in] config Global configuration dictionary.
+     * \param[in] link_index Map of link indices.
      */
-    void configure(const Dictionary& config) {
+    void configure(const Dictionary& config,
+                   std::map<std::string, int> link_index) {
```

Comment:
Argh, adding the map of robot links to params is not ideal :grimacing: It is rather a sign that the list we are building in there should be built by the caller instead.

Can you find a way to avoid passing this here?